input/includes/code-systems.md

```diff
@@ -0,0 +1,236 @@
+<table border="1" cellspacing="0" cellpadding="0" width="100%">
+<thead>
```

Comment:
I do not recommend this view for AU Core at this time.

Reply:
Thanks @dt-r, my apologies but committed this file by mistake. It is not used by the Terminology page. All content (Value Sets and Code Systems) is terminology.md.